common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -181,7 +181,7 @@ def xml_store_config(data_dir, course_dirs=None):
 TEST_DATA_MOCK_MODULESTORE = mixed_store_config(mkdtemp(), {}, include_xml=False)
 
 
-class ModuleStoreTestCase(TestCase):
+class ModuleStoreTestCase(TransactionTestCase):
```

Comment:
This will cause all cms/lms test cases which inherit from this to do the extra work of database cleanup and reloading fixtures for every test and will slow down the test suite (though I am not sure by how much).

It think it would be nicer to rename this to `ModuleStoreTransactionTestCase` and then create a new class with the original name:
```
class ModuleStoreTestCase(ModuleStoreTestCase, TestCase)
```

and then change the test cases which need the extra isolation to use the `ModuleStoreTransactionTestCase`.